frontend/www/js/omegaup/components/course/ViewProgress.vue

```diff
@@ -415,13 +415,34 @@ export default class CourseViewProgress extends Vue {
 }
 </script>
 
-<style scoped>
-.panel-body {
+<style lang="scss" scoped>
+.sticky-offset {
+  top: 4rem;
+}
+
+.table-fixed {
+  max-height: 80vh;
   overflow: auto;
-  white-space: nowrap;
 }
 
-.sticky-offset {
-  top: 4rem;
+.table-fixed thead th {
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  background: white;
+}
+
+.table-fixed tbody /deep/ th {
+  position: sticky;
+  left: 0;
+  background: white;
+  z-index: 1;
+}
+
+.table-fixed thead th:first-child {
+  position: sticky;
+  left: 0;
+  background: white;
+  z-index: 2;
 }
```

Comment:
```suggestion

  thead {
    th {
      position: sticky;
      top: 0;
      z-index: 1;
      background: white;
    }
    th:first-child {
      z-index: 2;
    }
  }

  tbody /deep/ th {
    position: sticky;
    left: 0;
    background: white;
    z-index: 1;
  }
}
```

se puede usar la sintaxis scss?